### DISCUSSION TITLE
版本号为20241006 请教一下你们的MPV播放器 在 geometry≠100% 前提下打开1080P以上的视频 窗口不会居中打开而是在屏幕外面

### DISCUSSION THREAD

**Discussion by Lonely233233** [hooke007/mpv_PlayKit#509] [mpv 通用]
MPV播放器打开的窗口会在屏幕以外

**Comment by hooke007** [comment]
https://github.com/shinchiro/mpv-winbuild-cmake/issues/707

**Reply by Lonely233233** [reply]
> [shinchiro/mpv-winbuild-cmake#707](https://github.com/shinchiro/mpv-winbuild-cmake/issues/707)

geometry=60% autofit-larger=60% 好像就没有这个问题了

**Reply by zhongfly** [reply]
https://github.com/mpv-player/mpv/pull/14938
感觉是这个pr导致的，之后对于Windows的问题做了几次修复。建议试试今天的最新版？

**Reply by hooke007** [reply]
Trace https://github.com/mpv-player/mpv/issues/15043
